exampleCourse/questions/demoAutograderSquare/question.html

```diff
@@ -0,0 +1,23 @@
+<pl-question-panel>
+  <p>
+    This is a simple example question that provides a variable to the student and expects one in return.  A list of variables for the user is defined in the HTML parameter <code>names_for_user</code>, and the student can only access these from the setup code.  Similarly, a list of variables from the user can be defined in <code>names_from_user</code>.  These two parameters control access to variables in the setup and the user code, and are also used to populate the variable tables below.
```

Comment:
I'm not sure this is an HTML parameter?

```suggestion
    This is a simple example question that provides a variable to the student and expects one in return.  A list of variables for the user is defined in the parameter <code>names_for_user</code>, and the student can only access these from the setup code.  Similarly, a list of variables from the user can be defined in <code>names_from_user</code>.  These two parameters control access to variables in the setup and the user code, and are also used to populate the variable tables below.
```